CONTRIBUTING.md

```diff
@@ -49,44 +86,18 @@ If the answer to either of those two questions are "yes", then you're probably d
 eXist follows a Semantic Versioning scheme, this is further documented in the [eXist Versioning Scheme and Release Process](exist-versioning-release.md) document.
 
 ### Porting during Release Candidate development phase
-When developing both a stable release and a release-candidate in parallel, this requires commits to be both back- and forward-ported until the release-candidate has become the next stable release.
+When developing one of more stable release lines and/or a release-candidate in parallel, this may require commits to be both back- and forward-ported until the release-candidate has become the next stable release.
 
-Under these circumstance pull-request should happen in threes. In normal times, contributors would:
--   open a `my-feature-branch` of `develop`
--   commit their changes
--   open a Pull Request from `my-feature-branch` into `develop`  
-
-This is still the default mode, however, when a release candidate is under development, the following two additional steps are necessary:
+Under these circumstance pull-request for the same purpose may be opened multiple times against different `develop`* branches
```

Comment:
s/Under these circumstance pull-request/In this circumstance a pull request

Reply:
Thanks @joewiz I pushed a commit to address this.